crates/luajit-bindings/src/ffi.rs

```diff
@@ -152,7 +152,10 @@ extern "C" {
     pub fn lua_typename(L: *mut lua_State, tp: c_int) -> *const c_char;
 
     // Lua auxiliary library.
-
+    
+    // https://www.lua.org/manual/5.3/manual.html#lua_gettable
+    pub fn lua_gettable(L: *mut lua_State, tp: c_int):
```

Comment:
the functions are ordered alphabetically, with the functions in the auxiliary library going after the ones in the API. Please reorder `lua_gettable` accordingly.